cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -4728,8 +4729,9 @@ macro(build_awssdk)
                                               "10")
     # Workaround for https://github.com/aws/aws-sdk-cpp/issues/1750
     set(AWSSDK_PATCH_COMMAND "sed" "-i.bak" "-e" "s/\"-Werror\"//g"
-                             "<SOURCE_DIR>/cmake/compiler_settings.cmake")
+                             "<SOURCE_DIR>/cmake/compiler_settings.cmake" "&&")
```

Comment:
Can we remove this patch?
It seems that 1.10.55 includes the fix for https://github.com/aws/aws-sdk-cpp/issues/1750 .

Reply:
done